[2022-02-21 16:53:13,336] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 16:53:13,353] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 16:53:13,353] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:13,353] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:13,353] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:13,365] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-16 00:00:00+00:00
[2022-02-21 16:53:13,373] {standard_task_runner.py:52} INFO - Started process 90 to run task
[2022-02-21 16:53:13,376] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-16T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmphmqtf4rg', '--error-file', '/tmp/tmpgcuivzji']
[2022-02-21 16:53:13,376] {standard_task_runner.py:77} INFO - Job 19: Subtask get_path
[2022-02-21 16:53:13,437] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:13,491] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-16T00:00:00+00:00
[2022-02-21 16:53:13,492] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:13,492] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:13,504] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220116T000000, start_date=20220221T165313, end_date=20220221T165313
[2022-02-21 16:53:13,549] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:13,591] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:01:55,872] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 17:01:55,886] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 17:01:55,886] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:55,886] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:01:55,886] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:55,899] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-16 00:00:00+00:00
[2022-02-21 17:01:55,904] {standard_task_runner.py:52} INFO - Started process 88 to run task
[2022-02-21 17:01:55,907] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-16T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpfkhvpyq5', '--error-file', '/tmp/tmpyrne48vg']
[2022-02-21 17:01:55,908] {standard_task_runner.py:77} INFO - Job 17: Subtask get_path
[2022-02-21 17:01:55,962] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:01:56,017] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-16T00:00:00+00:00
[2022-02-21 17:01:56,018] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:01:56,041] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220116T000000, start_date=20220221T170155, end_date=20220221T170156
[2022-02-21 17:01:56,079] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:01:56,110] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:39,693] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 19:10:39,701] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [queued]>
[2022-02-21 19:10:39,701] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:39,701] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:39,701] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:39,711] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-16 00:00:00+00:00
[2022-02-21 19:10:39,717] {standard_task_runner.py:52} INFO - Started process 88 to run task
[2022-02-21 19:10:39,719] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-16T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpgritkxs9', '--error-file', '/tmp/tmp28j1wjw4']
[2022-02-21 19:10:39,719] {standard_task_runner.py:77} INFO - Job 17: Subtask get_path
[2022-02-21 19:10:39,754] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-16T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:39,790] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-16T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-16T00:00:00+00:00
[2022-02-21 19:10:39,790] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:39,803] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220116T000000, start_date=20220221T191039, end_date=20220221T191039
[2022-02-21 19:10:39,850] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:39,868] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
